src/system/resources/app/app.py

```diff
@@ -20,37 +20,49 @@ class AppResource(Resource):
         'min_support_version': fields.String,
         'port': fields.Integer,
         **service_fields,
-        'name': fields.String,
-        'created_at': fields.String,
-        'browser_download_url': fields.String
+        'browser_download_url': fields.String,
+        'latest_version': fields.String
     }
 
     @classmethod
     @marshal_with(fields)
     def get(cls):
         try:
-            return cls.get_installed_apps_stat()
+            parser = reqparse.RequestParser()
+            parser.add_argument('browser_download_url', type=inputs.boolean, default=False)
+            parser.add_argument('latest_version', type=inputs.boolean, default=False)
+            args = parser.parse_args()
+            browser_download_url = args['browser_download_url']
+            latest_version = args['latest_version']
+            return cls.get_installed_apps_stat(browser_download_url, latest_version)
         except Exception as e:
             abort(501, message=str(e))
 
     @classmethod
-    def get_installed_apps_stat(cls):
+    def get_installed_apps_stat(cls, browser_download_url, latest_version):
         installed_apps = []
         for installable_app in inheritors(InstallableApp):
-            installed_apps.append(cls.get_installed_app_stat(installable_app()))
+            installed_apps.append(cls.get_installed_app_stat(installable_app(),browser_download_url,latest_version))
         return installed_apps
 
     @classmethod
-    def get_installed_app_stat(cls, app: InstallableApp) -> dict:
+    def get_installed_app_stat(cls, app: InstallableApp, browser_download_url, latest_version) -> dict:
         if systemctl_installed(app.service_file_name):
             details: dict = get_installed_app_details(app)
             if details:
                 app = get_app_from_service(details['service'], details['version'])
                 app_setting = current_app.config[AppSetting.FLASK_KEY]
-                browser_download_url = {}
-                try:
-                    browser_download_url = app.get_download_link(app_setting.token, True)
-                except Exception as e:
-                    logger.error(str(e))
-                return {**details, 'is_installed': True, **browser_download_url}
+                __browser_download_url = {}
+                __latest_version = None
```

Comment:
`__var` is used for class attributes for mangling the attributes --which needed to be protected on child classes. So use `_var` for internal use.